R/bayes_analysis.R

```diff
@@ -458,35 +494,37 @@ stemplot.df <- function(c,x) {
 
 
 
-### leverage
-
+#' leverage
```

Comment:
What is "leverage"? I could not work out what this function does from that description

Reply:
I have made it more detailed, but it isn't within the remit of the program to explain what leverage is. We don't explain most other statistical terms.